posts/aws-and-image-builder/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Customize AWS cloud images with Image Builder and Packit
+date: 2024-01-22
+authors: mmassari
+tags:
+  - image-builder
+---
+
+Have you ever wanted to bring your pull request changes in a cloud image easily?
+How easy is it? Just comment on your pull request with `/packit vm-image-build`.
```

Comment:
```suggestion
Curious about how easy it can be? With Packit, it can be just about commenting on your pull request with `/packit vm-image-build`.
```